lms/product/product.py

```diff
@@ -47,8 +47,12 @@ class Settings:
     groups_enabled: bool = False
     """Is the course groups feature enabled"""
 
+    create_lineitem: bool = False
+    """Create lineitem on assignment configuration"""
```

Comment:
I don't get what this setting is for? This looks specific to the implementation of the misc plugin for D2L, rather than a general thing.

Why would we ever turn this off for D2L or on for anyone else?